cmd/main.go

```diff
@@ -1,5 +1,33 @@
 package main
 
+/**
+                            ▒▒▒▒▒▒▒▒▒▒▒▒                    
+                          ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒                  
+        ░░░░░░░░░░        ▒▒▒▒        ▒▒▒▒▒▒                
+      ░░░░░░░░░░░░░░      ▒▒▒▒          ▒▒▒▒                
+      ▒▒▒▒▒▒▒▒▒▒▒▒▒▒      ▒▒▒▒▒▒        ▒▒▒▒                
+  ░░░░▒▒▒▒▒▒▒▒▒▒▒▒▒▒░░░░    ▓▓▒▒▒▒      ▒▒▒▒                
+░░░░░░░░▒▒▒▒▒▒▒▒▒▒░░░░░░░░    ▒▒▒▒      ▒▒▒▒                
+▒▒░░░░░░░░░░░░░░░░░░░░░░▒▒    ▒▒▒▒        ▒▒▒▒              
+▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒    ▒▒▒▒        ▒▒▒▒              
+░░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░░    ▒▒▒▒        ▒▒▒▒              
+░░░░░░▒▒▒▒▒▒▒▒▒▒▒▒▒▒░░░░░░  ▒▒▒▒▒▒        ▒▒▒▒              
+░░░░░░░░░░░░░░░░░░░░░░░░▒▒▒▒▒▒▒▒          ▒▒▒▒              
+░░░░░░░░░░░░░░░░░░░░░░░░▒▒▒▒▒▒            ▒▒▒▒              
+░░░░░░░░░░░░░░░░░░░░░░░░░░                ▒▒▒▒              
+░░░░░░░░░░░░░░░░░░░░░░░░░░                  ▒▒▓▓            
+░░░░░░░░░░░░░░░░░░░░░░░░░░                  ▒▒▒▒            
+░░░░░░░░░░░░░░░░░░░░░░░░░░                  ▒▒▒▒            
+▒▒░░░░░░░░░░░░░░░░░░░░░░▒▒                  ▒▒▒▒            
+▒▒▒▒▒▒▒▒░░░░░░░░░░▒▒▒▒▒▒▒▒                  ▒▒▒▒            
+██▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒██                ▒▒▒▒▒▒▒▒          
+████    ▒▒▒▒▒▒▒▒▒▒    ████              ▒▒▒▒▒▒▒▒▒▒▒▒        
+          ████                            ░░░░░░░░░░░░░░░░░░
+                                        ░░░░░░░░░░░░░░░░░░  
+                                            ░░░░░░░░░░      
+
+*/
```

Comment:
love this mate, think I should add this to the console output when you run it :rocket: